_includes/banner.html

```diff
@@ -1,10 +1,11 @@
 <div class="row banner">
-    <div class="row">
-        <h1 class="text-lg-center">Giving you a voice in the #OpenSource Tech community to inspire others</h1>
-        <div class="jumbotron">
+    <p></p>
```

Comment:
ok, I like the idea, but I would do this with **CSS**.

In the file `css/custom.css` you could add a `margin` to the `h1` tag using a descriptive selector like 

```css
div.banner h1.text-lg-center {
   margin-top: 2rem;
}
```